### PR TITLE
cloud: add 20240103 release notes

### PR DESCRIPTION
**Pull request by ran-huang**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/release-notes-2023.md
Comment (qiancai) [inline tidb-cloud/release-notes-2023.md:1]: 需要加到 TOC 里

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 1]
```diff
@@ -1,1004 +1,27 @@
...
+    For more information, see [Organization SSO Authentication](/tidb-cloud/tidb-cloud-org-sso-authentication.md).
 
-- Enhance the connection experience for [TiDB Serverless](/tidb-cloud/select-cluster-tier.md#tidb-serverless).
+- Upgrade the default TiDB version of new [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated) clusters from [v7.1.1](https://docs.pingcap.com/tidb/v7.1/release-7.1.1) to [v7.5.0](https://docs.pingcap.com/tidb/stable/release-7.5.0).
 
-    Refine the **Connect** dialog interface to offer TiDB Serverless users a smoother and more efficient connection experience. In addition, TiDB Serverless introduces more client types and allows you to select the desired branch for connection.
+- The feature of [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated) dual region backup is now in General Availability (GA).
 
-    For more information, see [Connect to TiDB Serverless](/tidb-cloud/connect-via-standard-connection-serverless.md).
+    By using this feature, you can replicate backups across geographic regions within AWS/Google Cloud. This feature provides an additional layer of data protection and disaster recovery capabilities.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-release-notes.md:25]:
```suggestion
    By using this feature, you can replicate backups across geographic regions within AWS or Google Cloud. This feature provides an additional layer of data protection and disaster recovery capabilities.
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 2]
```diff
@@ -1,1004 +1,27 @@
...
+    With this feature, you can seamlessly integrate TiDB Cloud with any identity provider (IdP) using [Security Assertion Markup Language (SAML)](https://en.wikipedia.org/wiki/Security_Assertion_Markup_Language) or [OpenID Connect (OIDC)](https://openid.net/developers/how-connect-works/).
 
-**Console changes**
+    For more information, see [Organization SSO Authentication](/tidb-cloud/tidb-cloud-org-sso-authentication.md).
 
-- Enhance the connection experience for [TiDB Serverless](/tidb-cloud/select-cluster-tier.md#tidb-serverless).
+- Upgrade the default TiDB version of new [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated) clusters from [v7.1.1](https://docs.pingcap.com/tidb/v7.1/release-7.1.1) to [v7.5.0](https://docs.pingcap.com/tidb/stable/release-7.5.0).
 
-    Refine the **Connect** dialog interface to offer TiDB Serverless users a smoother and more efficient connection experience. In addition, TiDB Serverless introduces more client types and allows you to select the desired branch for connection.
+- The feature of [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated) dual region backup is now in General Availability (GA).
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-release-notes.md:23]:
```suggestion
- The dual region backup feature for [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated) is now in General Availability (GA).
```

File: tidb-cloud/tidb-cloud-faq.md [hunk 3]
```diff
@@ -41,7 +41,7 @@ No.
 
 ### What versions of TiDB are supported on TiDB Cloud?
 
-- Starting from October 31, 2023, the default TiDB version for new TiDB Dedicated clusters is v7.1.2.
+- Starting from October 31, 2023, the default TiDB version for new TiDB Dedicated clusters is v7.5.0.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-faq.md:44]:
```suggestion
- Starting from January 3, 2024, the default TiDB version for new TiDB Dedicated clusters is v7.5.0.
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 4]
```diff
@@ -1,1004 +1,27 @@
...
+- Support [Organization SSO](https://tidbcloud.com/console/preferences/authentication) to streamline enterprise authentication processes.
 
-    For more information, see [Changefeed states](/tidb-cloud/changefeed-overview.md#changefeed-states).
+    With this feature, you can seamlessly integrate TiDB Cloud with any identity provider (IdP) using [Security Assertion Markup Language (SAML)](https://en.wikipedia.org/wiki/Security_Assertion_Markup_Language) or [OpenID Connect (OIDC)](https://openid.net/developers/how-connect-works/).
 
-**Console changes**
+    For more information, see [Organization SSO Authentication](/tidb-cloud/tidb-cloud-org-sso-authentication.md).
 
-- Enhance the connection experience for [TiDB Serverless](/tidb-cloud/select-cluster-tier.md#tidb-serverless).
+- Upgrade the default TiDB version of new [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated) clusters from [v7.1.1](https://docs.pingcap.com/tidb/v7.1/release-7.1.1) to [v7.5.0](https://docs.pingcap.com/tidb/stable/release-7.5.0).
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-release-notes.md:21]:
```suggestion
- Upgrade the default TiDB version of new [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated) clusters from [v7.1.1](https://docs.pingcap.com/tidb/v7.1/release-7.1.1) to [v7.5.0](https://docs.pingcap.com/tidb/v7.5/release-7.5.0).
```

File: tidb-cloud/release-notes-2023.md [hunk 5]
```diff
@@ -0,0 +1,1003 @@
...
+
+## October 25, 2023
+
+**General changes**
+
+- [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated) supports dual region backup (beta) on Google Cloud.
+
+    TiDB Dedicated clusters hosted on Google Cloud work seamlessly with Google Cloud Storage. Similar to the [Dual-regions](https://cloud.google.com/storage/docs/locations#location-dr) feature of Google Cloud Storage, the pair of regions that you use for the dual-region in TiDB Dedicated must be within the same multi-region. For example, Tokyo and Osaka are in the same multi-region `ASIA` so they can be used together for dual-region storage.
+
+    For more information, see [Back Up and Restore TiDB Dedicated Data](/tidb-cloud/backup-and-restore.md#turn-on-dual-region-backup-beta).
```
Comment (ran-huang) [inline tidb-cloud/release-notes-2023.md:97]:
```suggestion
    For more information, see [Back Up and Restore TiDB Dedicated Data](/tidb-cloud/backup-and-restore.md#turn-on-dual-region-backup).
```

File: tidb-cloud/release-notes-2023.md [hunk 6]
```diff
@@ -0,0 +1,1003 @@
...
+
+    After a successful 10-month beta trial, the feature of streaming data change logs from TiDB Cloud to Apache Kafka becomes generally available. Streaming data from TiDB to a message queue is a common need in data integration scenarios. You can use Kafka sink to integrate with other data processing systems (such as Snowflake) or support business consumption.
+
+    For more information, see [Changefeed overview](/tidb-cloud/changefeed-overview.md).
+
+## October 11, 2023
+
+**General changes**
+
+- Support [dual region backup (beta)](/tidb-cloud/backup-and-restore.md#turn-on-dual-region-backup-beta) for [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated) clusters deployed on AWS.
```
Comment (ran-huang) [inline tidb-cloud/release-notes-2023.md:109]:
```suggestion
- Support [dual region backup (beta)](/tidb-cloud/backup-and-restore.md#turn-on-dual-region-backup) for [TiDB Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-dedicated) clusters deployed on AWS.
```

**Pull request merged**